README.md

```diff
@@ -88,3 +88,16 @@ Make your changes.
 Commit your changes (git commit -m 'Add some feature').
 Push to the branch (git push origin feature/your-feature-name).
 Open a pull request.
+
+---
+
+## Our Awesome Contributors 🚀
+
+- Thank you for your valuable contribution to this project. We truly appreciate your skills and dedication. Keep up the great work!
+- Make sure you show some love by giving ⭐ to our repository.
+
+<div align="left">
+  <a href="https://github.com/AlfiyaSiddique/TastyTrails">
```

Comment:
bro.. why did you add your repo link here ??

Reply:
Ohh, I am sorry. I was testing it out and forgot to change the repo link!